content/docs/command-reference/import-url.md

```diff
@@ -11,7 +11,7 @@ etc.), and download it to the local project, or make a copy in
 
 ```usage
 usage: dvc import-url [-h] [-q | -v] [-j <number>] [--file <filename>]
-                      [--no-exec] [--to-remote] [-r <name>]
+                      [--no-exec | --no-download] [--to-remote] [-r <name>]
                       [--desc <text>]
                       url [out]
```

Comment:
This is now too long when rendered. See https://dvc.org/doc/command-reference/import-url

💅🏼 but let's regroup the args? Same for the other 2 files.